sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -6152,7 +6154,9 @@ private <T> Flux<FeedResponse<T>> nonDocumentReadFeedInternal(
         Integer maxItemCount = ModelBridgeInternal.getMaxItemCountFromQueryRequestOptions(nonNullOptions);
         int maxPageSize = maxItemCount != null ? maxItemCount : -1;
 
-        logger.warn("ReadFeed call made for ResourceType : [{}] and ResourceLink : [{}] with pageSize : [{}] for UserAgent : [{}]", resourceType, resourceLink, maxPageSize, this.userAgentContainer.getUserAgent());
+        if (ResourceType.PartitionKeyRange.equals(resourceType)) {
```

Comment:
maybe better to change around `int maxPageSize = maxItemCount != null ? maxItemCount : -1;`. I think null indicates no override

Reply:
Fixed